### PR TITLE
[t1784] project settings editor

### PR DESCRIPTION
**Pull request by secretrobotron**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/df0e6753b54b41d/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/df0e6753b54b41d/output.txt

Total script time: 1.07 mins

Server is running on http://pr849.butyr.org/templates/basic/
Run the Test Suite: http://pr849.butyr.org/test/

File: css/buttons.less [hunk 1]
```diff
@@ -21,6 +21,32 @@
   &:hover {
     cursor: pointer;
   }
+
+  &[disabled] {
+    .gradient( "center top", #c1c1c1, #c0c0c0, false );
+  }
+}
+
+.butter-btn-green {
```
Comment (k88hudson) [inline css/buttons.less]: Why is this getting added?

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @cadecairos received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/04b3d26ba2e33ea/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @cadecairos received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/b5193e09caaaab5/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/04b3d26ba2e33ea/output.txt

Total script time: 0.75 mins

Server is running on http://pr849.butyr.org/templates/basic/
Run the Test Suite: http://pr849.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/b5193e09caaaab5/output.txt

Total script time: 1.13 mins

**Pull request merged**